src/Admin/Views/Providers/Organizations.cshtml

```diff
@@ -6,7 +6,14 @@
             <table class="table table-striped table-hover">
                 <thead>
                     <tr>
-                        <th>Name</th>
+                        <th style="width: 50%;">Name</th>
+                        <th style="width: 50%;">Status</th>
+                        <th>
+                            <div class="float-right text-nowrap">
+                                <a asp-controller="Organizations" asp-action="Create" asp-route-providerId="@Model.Provider.Id" class="btn btn-sm btn-primary">New Organization</a>
+                                <a asp-controller="Providers" asp-action="AddExistingOrganization" asp-route-id="@Model.Provider.Id" class="btn btn-sm btn-outline-primary">Add Existing Organization</a>
+                            </div>
+                        </th>
```

Comment:
Just noticed this when reviewing one of your PRs. These buttons should only display for Resellers. We can't create or add existing orgs to an MSP from here because it needs a key exchange, that's managed via the Provider Portal.

Reply:
That makes sense, I'll make that change on another PR. Thanks!

Reply:
It is addressed [here](https://github.com/bitwarden/server/pull/2723).